src/blueprints/tv2/action-factories/tv2-graphics-action-factory.ts

```diff
@@ -32,7 +73,7 @@ export class Tv2GraphicsActionFactory {
       name: 'Gfx All Out',
       duration,
       timelineObjects: [
-        this.graphicsTimelineObjectFactory.createAllOutGraphicsTimelineObject(blueprintConfiguration, duration)
+        this.vizTimelineObjectFactory.createAllOutGraphicsTimelineObject(blueprintConfiguration, duration)
```

Comment:
Because the `Tv2GraphicsActionFactory` takes in two implementation for `GraphicsTimelineObjectFactory` - One for Viz and one for CasparCG.
The CasparCG doesn't have an implementation for the timeline object needed in this context, thus it is always the viz version that should be called.

This will be fixed during SOF-1696, which i am hoping to get to during this sprint, if others haven't already done so.